services/harmony/test/parameters/concatenation.ts

```diff
@@ -182,52 +182,4 @@ describe('testing concatenation', function () {
       });
     });
   });
-
```

Comment:
I pulled this set of testing because there is no NetCDF-to-Zarr workflow. I'm not sure if this is a cause for concern, though, as it was testing a workflow where the default concatenation behaviour is to not, unlike concise.